src/main/java/lotto/domain/PrizeResult.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PrizeResult {
+    private final Map<MatchInfo, Integer> prizeResultInfoList = new HashMap<>();
+
+    public PrizeResult(){
+        for(MatchInfo matchInfo : MatchInfo.values()){
+            prizeResultInfoList.put(matchInfo, 0);
+        }
+    }
```

Comment:
[컨벤션](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace)이 지켜지지 않은 코드가 많은 것 같아요
세이브 또는 커밋 전에 ide 의 reformat code 기능을 활용해 보시면 좋을 것 같아요
```suggestion
    public PrizeResult() {
        for (MatchInfo matchInfo : MatchInfo.values()) {
            prizeResultInfoList.put(matchInfo, 0);
        }
    }
```